sdk/test-utils/recorder-new/src/index.ts

```diff
@@ -6,3 +6,4 @@ export { TestProxyHttpClientCoreV1 } from "./core-v1-recorder";
 export { relativeRecordingsPath } from "./utils/relativePathCalculator";
 export { SanitizerOptions, RecorderStartOptions } from "./utils/utils";
 export { NoOpCredential } from "./recorderAADCredential";
+export { env } from "@azure-tools/test-recorder";
```

Comment:
Why do we need to export env from the `old` recorder?

Reply:
Think this should be exporting from `./utils/env` instead

Reply:
I can redeclare env.